src/Employment/WorkContract_v1.0.py

```diff
@@ -0,0 +1,340 @@
+import datetime
+from datetime import date
+from enum import Enum
+from typing import List, Optional
+
+from definition_tooling.converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class WorkContractRequest(CamelCaseModel):
+    end_user_authentication_token: str = Field(
+        ..., description="The end user authentication token"
+    )
+
+
+class EmployerInfo(CamelCaseModel):
+    name: str = Field(
+        ...,
+        title="Name",
+        max_length=250,
+        description="The official name of the employer",
+        examples=["Staffpoint Oy"],
+    )
+    business_id: str = Field(
+        ...,
+        title="Business ID",
+        max_length=250,
+        description="The business ID of the employer",
+        examples=["2492090-1"],
+    )
+    street_name: str = Field(
+        ...,
+        title="Street Name",
+        max_length=250,
+        description="The street name of the employer contact address",
+        examples=["Ruoholahdenkatu 17 A 6"],
+    )
+    postal_code: str = Field(
+        ...,
+        title="Postal Code",
+        max_length=10,
+        description="The postal code of the employer address",
+        examples=["00180"],
+    )
+    city: str = Field(
+        ...,
+        title="City",
+        max_length=250,
+        description="The city of the employer address",
+        examples=["Helsinki"],
+    )
+    signature_date: date = Field(
+        ...,
+        title="Signature Date",
+        description="The date of the employer signature for the work contract",
+        examples=[datetime.datetime(2023, 8, 25)],
+    )
+
+
+class EmployeeInfo(CamelCaseModel):
+    name: str = Field(
+        ...,
+        title="Name",
+        max_length=250,
+        description="The name of the employee",
+        examples=["Tom Williams"],
+    )
+    street_address: str = Field(
+        ...,
+        title="Street Address",
+        max_length=40,
+        description="The street name of the employee contact address",
+        examples=["49 Featherstone Street"],
+    )
+    postal_code: str = Field(
+        ...,
+        title="Postal Code",
+        max_length=10,
+        description="The postal code of the employee address",
+        examples=["EC1Y 8SY"],
+    )
+    city: str = Field(
+        ...,
+        title="City",
+        max_length=40,
+        description="The city of the employee address",
+        examples=["London"],
+    )
+    country: Optional[str] = Field(
+        ...,
+        title="Country",
+        max_length=3,
```

Comment:
If we want this to be a 3 letter code for the country, should it possibly have both min_length and max_length set to 3?